src/main/java/com/github/mauricioaniche/ck/CKMethodResult.java

```diff
@@ -41,13 +44,14 @@
 	//All direct invocations of methods of the same class
 	private Set<String> methodInvocationsLocal;
 	//All indirect invocations of methods of the same class
-	private Set<String> methodInvocationsIndirectLocal;
+	private Map<String, Set<String>> methodInvocationsIndirectLocal;
```

Comment:
why is it a map of string and set of strings now?

Reply:
I use the structure of the invocations, in order to improve the performance of the algorithm.